packages/profile/src/views/Profile.jsx

```diff
@@ -56,33 +57,6 @@ const change = (dispatch, type, e) => {
 
 const value = (state, type) => state[type]
 
-const save = async (state, dispatch, setSubmitting, e) => {
-  e.preventDefault()
-  setSubmitting(true)
-
-  const { profiles, members, memberId, profileId } = state
-  const data = Object.keys(state)
-    .filter(key => !['profiles', 'members', 'profileId', 'wallet'].includes(key))
-    .reduce((acc, key) => Object.assign(acc, { [key]: state[key] }), {})
-  console.log(data)
-
-  if (profileId && memberId) {
-    const patched = await profiles.patch(profileId, data)
-    console.log('patched', patched)
-    setSubmitting(false)
-    return patched
-  }
-
-  const profile = await profiles.create(data)
-  console.log('new', profile)
-  dispatch({ type: 'profileId', payload: profile.id })
-
-  const member = await members.patch(profile.data.memberId, { profileId: profile.id })
-  console.log(member)
-  setSubmitting(false)
-  // dispatch({ type: 'created', payload: updated })
-}
-
```

Comment:
(as mentioned in PR description) I moved this into the `Profile` component because otherwise I'd have to pass in a bunch more `useState` variables.

Reply:
Oh, I wasn't advocating for using the state hooks instead of the reducer pattern. I'm fine with reducer; I just felt that the non-React-specific JS being written in a more functional-programming style may take longer for devs coming from the conventional web-dev-JS / OOP-style background to wrap their heads around. But as I've never worked with React forms without relying on a library that manages form state, I don't have a different pattern to suggest at this time.

I just used state hooks here because I wasn't sure whether we were trying to reserve the existing `state` object for form fields. But I can add these into the existing `state` pattern.